### PR TITLE
Feature/mapa 5 create scheleton

### PR DESCRIPTION
**Pull request by claudiodamico**
Uploading rewiews

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Ignoring files with full path.
+# This matches files in the root directory and subdirectories too.
+# i.e. otherfile.ext will be ignored anywhere on the tree.
+dir/otherdir/file.ext
+otherfile.ext
+
+# Ignoring directories
+# Both the directory itself and its contents will be ignored.
+bin/
+gen/
```
Comment (gpgpublickey) [inline .gitignore:19]: please add these directories

[Dd]ebug/
[Dd]ebugPublic/
[Rr]elease/
[Rr]eleases/
x64/
x86/
[Ww][Ii][Nn]32/
[Aa][Rr][Mm]/
[Aa][Rr][Mm]64/
bld/
[Bb]in/
[Oo]bj/
[Ll]og/
[Ll]ogs/

and files:
*.cache

File: Backend_Mapanimal/Controllers/RaceController.cs [hunk 2]
```diff
@@ -3,47 +3,48 @@
...
     [ApiController]
     public class RaceController : ControllerBase
     {
-        private readonly ILogger<RaceController> logger;
+        private readonly ILogger<RaceController> _logger;
 
         public RaceController(ILogger<RaceController> logger)
         {
-            this.logger = logger;
+            this._logger = logger;
```
Comment (gpgpublickey) [inline Backend_Mapanimal/Controllers/RaceController.cs]: this is not required

File: Backend_Mapanimal/Entities/Race.cs [hunk 3]
```diff
@@ -2,8 +2,8 @@
 {
     public class Race
     {
-        public int Id { get; set; }
+        private int Id { get; set; }
```
Comment (gpgpublickey) [inline Backend_Mapanimal/Entities/Race.cs]: just the setters must be private, in another way we can't read values from entities.
ex:
`public int Id { get; private set; }`

**Review by gpgpublickey — CHANGES_REQUESTED** [review]
Thx Claudio, just a couple of additional changes and we will have it ready to merge

File: .gitignore (hunk 1 above)
Comment (claudiodamico) [reply .gitignore:19]: resolved

File: Backend_Mapanimal/Controllers/RaceController.cs (hunk 2 above)
Comment (claudiodamico) [reply Backend_Mapanimal/Controllers/RaceController.cs]: resolved

File: Backend_Mapanimal/Entities/Race.cs (hunk 3 above)
Comment (claudiodamico) [reply Backend_Mapanimal/Entities/Race.cs]: resolved

**Comment by claudiodamico** [comment]
review resolved

File: .gitignore (hunk 1 above)
Comment (gpgpublickey) [reply .gitignore:19]: *.cache is missing, please add it

File: Backend_Mapanimal/Entities/Race.cs (hunk 3 above)
Comment (gpgpublickey) [reply Backend_Mapanimal/Entities/Race.cs]: I can't see your changes, maybe you forgot to commit them

**Review by gpgpublickey — APPROVED** [review]

**Pull request merged**